{"timestamp": "2025-10-29 12:56:29,197", "level": "INFO", "logger": "src.client", "message": "POST https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=BUY&type=MARKET&quantity=0.001&timestamp=1761722789197&recvWindow=5000&signature=c6efa1724fb2aed16788dabfb77141ba719c109be5917d7fdcff82f5a99c56c6"}
{"timestamp": "2025-10-29 12:56:30,082", "level": "ERROR", "logger": "src.client", "message": "Order error: 401 {'code': -2015, 'msg': 'Invalid API-key, IP, or permissions for action'}"}
{"timestamp": "2025-10-29 12:56:30,083", "level": "ERROR", "logger": "__main__", "message": "Market order failed", "exc_info": "Traceback (most recent call last):\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\market_orders.py\", line 23, in main\n    res = client.create_order(symbol, side, 'MARKET', qty)\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\src\\client.py\", line 55, in create_order\n    r.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\myenv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=BUY&type=MARKET&quantity=0.001&timestamp=1761722789197&recvWindow=5000&signature=c6efa1724fb2aed16788dabfb77141ba719c109be5917d7fdcff82f5a99c56c6"}
{"timestamp": "2025-10-29 13:14:04,352", "level": "INFO", "logger": "src.client", "message": "POST https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=BUY&type=MARKET&quantity=0.001&timestamp=1761723844352&recvWindow=5000&signature=47f8cb5cbf02e93e5d64af24cbd5a12f0b2f7a98801f3d833dd3637f7c3ff0c0"}
{"timestamp": "2025-10-29 13:14:05,148", "level": "ERROR", "logger": "src.client", "message": "Order error: 401 {'code': -2015, 'msg': 'Invalid API-key, IP, or permissions for action'}"}
{"timestamp": "2025-10-29 13:14:05,149", "level": "ERROR", "logger": "__main__", "message": "Market order failed", "exc_info": "Traceback (most recent call last):\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\market_orders.py\", line 23, in main\n    res = client.create_order(symbol, side, 'MARKET', qty)\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\src\\client.py\", line 55, in create_order\n    r.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\myenv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=BUY&type=MARKET&quantity=0.001&timestamp=1761723844352&recvWindow=5000&signature=47f8cb5cbf02e93e5d64af24cbd5a12f0b2f7a98801f3d833dd3637f7c3ff0c0"}
{"timestamp": "2025-10-29 13:14:20,410", "level": "INFO", "logger": "src.client", "message": "POST https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=SELL&type=LIMIT&quantity=0.001&price=75000.0&timeInForce=GTC&timestamp=1761723860410&recvWindow=5000&signature=4d89a037b54b10bd2dc49dbb2ab48f411199794c65c7aa585b9d2f1d6e7d9e42"}
{"timestamp": "2025-10-29 13:14:21,271", "level": "ERROR", "logger": "src.client", "message": "Order error: 401 {'code': -2015, 'msg': 'Invalid API-key, IP, or permissions for action'}"}
{"timestamp": "2025-10-29 13:14:21,272", "level": "ERROR", "logger": "__main__", "message": "Limit order failed", "exc_info": "Traceback (most recent call last):\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\limit_orders.py\", line 26, in main\n    res = client.create_order(symbol, side, 'LIMIT', qty, price=price, timeInForce=args.time_in_force)\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\src\\client.py\", line 55, in create_order\n    r.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\HP15\\Desktop\\piyush_binance_bot\\myenv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=SELL&type=LIMIT&quantity=0.001&price=75000.0&timeInForce=GTC&timestamp=1761723860410&recvWindow=5000&signature=4d89a037b54b10bd2dc49dbb2ab48f411199794c65c7aa585b9d2f1d6e7d9e42"}
{"timestamp": "2025-10-29 13:48:46,948", "level": "INFO", "logger": "src.client", "message": "POST https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=BUY&type=MARKET&quantity=0.001&timestamp=1761725926948&recvWindow=5000&signature=c384bb72b0f9c30ae82be54d56e50d3f84383e7f701ebc0e551ed7b1b7a0575e"}
{"timestamp": "2025-10-29 13:48:49,029", "level": "ERROR", "logger": "src.client", "message": "Order error: 401 {'code': -2015, 'msg': 'Invalid API-key, IP, or permissions for action'}"}
{"timestamp": "2025-10-29 13:48:49,031", "level": "ERROR", "logger": "webapp", "message": "UI order error: 401 Client Error: Unauthorized for url: https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=BUY&type=MARKET&quantity=0.001&timestamp=1761725926948&recvWindow=5000&signature=c384bb72b0f9c30ae82be54d56e50d3f84383e7f701ebc0e551ed7b1b7a0575e"}
